eva_sub_cli/executables/xlsx2json.py

```diff
@@ -219,16 +194,17 @@ def get_biosample_object(self, data):
         scientific_name = self.xlsx_conf[SAMPLE][OPTIONAL_HEADERS_KEY_NAME][SCIENTIFIC_NAME_KEY]
 
         # BioSample expects any of organism or species field
-        data[SPECIES] = data[scientific_name]
+        if scientific_name in data:
+            data[SPECIES] = data[scientific_name]
```

Comment:
Is it worth having a mechanism that convert a excel spreadsheet into more than one BioSample field ? 
We could then provide this in the `spreadsheet2json_conf.yaml` as 
```yaml
Sample:
  header_row: 3
  optional:
    Scientific Name: [scientificName, species]
```

Reply:
Could do, it would only be used for this case though as far as I know - the name field goes outside of characteristics so has to be handled differently. I probably won't add it to this PR but we should keep it in mind.